UI/js/utils/utils.js

```diff
@@ -60,6 +60,83 @@ var dashMessageSeenUpdate = function() {
   sessionStorage.setItem('dashMessageSeen', true);
 };
 
+var statusResolver = function(status, count) {
+  var status, message, count;
+ 
+ switch (status) {
+  case 200:
+    message = "- all is well";
+    break;
+  case 400:
+    message = "- bad request";
+    break;
+  case 401:
+    message = "- not authorized";
+    break;
+  case 403:
+    message = "- forbidden";
+    break;
+  case 404:
+    message = "- not found";
+    break;
+  case 408:
+    message = "- request timeout";
+    break;
+  case 500:
+    message = "- internal Server Error";
+    break;
+  case 504:
+    message = "- gateway Timeout";
+    break;
+  case 666:
+    message = "- demonic possession";
+    break;
+  default:
+    message = "- status not accounted for";
+  }
+  return {
+    "status": status,
+    "message": message,
+    "count": count
+  }
+}
+
+
+var prepareToDos = function(result) {
+  var combinedTodosAndStatus = {
+    'status': {
+      'ctools': statusResolver(result.data.ctools.Meta.httpStatus, result.data.ctools.Result.length),
+      'canvas': statusResolver(result.data.canvas.Meta.httpStatus, result.data.canvas.Result.length)
+    },
+    'combinedToDos': []
+  }
+  var status = {}
+  var statusCTools = result.data.ctools.Meta.httpStatus;
+  var statusCTools = result.data.ctools.Meta.httpStatus;
+  var combinedTodos = result.data.ctools.Result.concat(result.data.canvas.Result);
+  $.each(combinedTodos, function() {
+    this.due_date_long = moment.unix(this.due_date_sort).format('dddd, MMMM Do YYYY, h:mm a');
+    this.due_date_short = moment.unix(this.due_date_sort).format('MM/DD');
```

Comment:
I notice there is a style change between variables using camelCase and under_score. Is this to differentiate between actual variables and request parameters like we do with Canvas Course Manager?